deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -250,25 +250,52 @@ def deploy_linkerd(settings):
 def deploy_postgres(settings):
     print_separator("postgres")
 
-    docker_build("postgres", settings)
-
-    restore_seqr_db_from_backup = settings.get("RESTORE_SEQR_DB_FROM_BACKUP")
-    reset_db = settings.get("RESET_DB")
-
-    if reset_db or restore_seqr_db_from_backup:
-        # Since pgdata is stored on a persistent volume, redeploying does not get rid of it. If any existing pgdata is
-        # present, even if the databases are empty, postgres will not fully re-initialize the database. This is
-        # good if you want to keep the data across a deployment, but problematic if you actually need to rest and
-        # reinitialize the db. Therefore, when the database needs to be fully reinitialized, delete pgdata
-        run_in_pod(get_pod_name("postgres", deployment_target=settings["DEPLOY_TO"]),
-                   "rm -rf /var/lib/postgresql/data/pgdata", verbose=True)
-
-    deploy_pod("postgres", settings, wait_until_pod_is_ready=True)
-
-    if restore_seqr_db_from_backup:
-        postgres_pod_name = get_pod_name("postgres", deployment_target=settings["DEPLOY_TO"])
-        _restore_seqr_db_from_backup(
-            postgres_pod_name, restore_seqr_db_from_backup, settings.get("RESTORE_REFERENCE_DB_FROM_BACKUP"))
+    with open('deploy/secrets/gcloud/postgres/{}/password'.format(settings['DEPLOY_TO'])) as f:
+        password = f.read()
+
+    sql_instance_name = 'postgres-{}'.format(settings['DEPLOYMENT_TYPE'])
+    run(' '.join([
+        'gcloud beta sql instances create', sql_instance_name,
+        '--database-version=POSTGRES_{}'.format(settings['POSTGRES_VERSION']),
+        '--root-password={}'.format(password),
+        '--project={}'.format(settings['GCLOUD_PROJECT']),
+        '--zone={}'.format(settings['GCLOUD_ZONE']),
+        '--availability-type=regional',
+        '--cpu=4', '--memory=26',
+        '--assign-ip',
+        '--backup',
+        '--maintenance-release-channel=production', '--maintenance-window-day=SUN', '--maintenance-window-hour=5',
+        '--require-ssl',
+        '--retained-backups-count=30',
+        '--storage-auto-increase',
+    ]), errors_to_ignore=['already exists'])
+
+    seqr_db_backup = settings.get('RESTORE_SEQR_DB_FROM_BACKUP')
+    reference_data_db_backup = settings.get('RESTORE_REFERENCE_DB_FROM_BACKUP')
```

Comment:
If I'm understanding correctly, these variables affect whether or not we perform a restore operation. I think maybe it might make sense to name the variables with a `_restore` suffix? My brain jumps to the conclusion that we're performing a backup with the current prefix. Not a huge deal, though.

Reply:
the variables contain the path to the backup file if we want to do a restore, and are null if we shouldn't do a restore. I agree that the current combination of settings variables called `RESTORE_X_DB_FROMBACKUP` being assigned to local variables called `X_db_backup` is confusing, but I think changing the local variable name would make the below code of `'gcloud sql import sql', sql_instance_name, X_db_backup_restore, '--database=X_db',` a little weird looking. But if you think thats still readable I'm happy to change it